app/models/node-addon.ts

```diff
@@ -0,0 +1,17 @@
+import { attr } from '@ember-data/model';
+
+import OsfModel from './osf-model';
+
+export default class NodeAddonModel extends OsfModel {
+    @attr('boolean') nodeHasAuth!: boolean;
+    @attr('boolean') configured!: boolean;
+    @attr('string') external_account_id!: string;
+    @attr('string') folder_id?: string;
+    @attr('string') folder_path?: string;
```

Comment:
Do we want to leave these snake_case? If so, we may need to override the default behavior in app/serializers/node-addon.ts

Reply:
Ha! Whoops.